services/auth-service/src/main/java/com/sourcefuse/jarc/services/authservice/enums/AuthErrorKeys.java

```diff
@@ -0,0 +1,24 @@
+package com.sourcefuse.jarc.services.authservice.enums;
+
+public enum AuthErrorKeys {
+  CODE_EXPIRED("Code Expired"),
```

Comment:
Should we not have them as numeric enums ? We will anyways need to use i18n for localisation